main.js

```diff
@@ -76,19 +93,22 @@ function changeImgColor(whiteImg, activeImg) {
 };
 
 
-function activateStudyColor() {
+function activateStudy() {
```

Comment:
Good call on renaming to better encompass what all is actually happening/will happen with this function.